src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -356,250 +110,82 @@ public void resetInitialLocalization() {
-      //     est -> {
-      //       visionEstPose.set(est.estimatedPose);
-      //       visionEstPoseTimestamp.set(est.timestampSeconds);
-
-      //       // Change our trust in the measurement based on the tags we can see
-      //       var estStdDevs = getEstimationStdDevs();
-
-      //       m_swerveDriveTrain.addVisionMeasurement(
-      //           est.estimatedPose.toPose2d(), est.timestampSeconds, estStdDevs);
-      //     });
-
-      // // if (cameraAHasPose && cameraBHasPose) {
-      //   poseAgreement = checkPoseAgreement(cameraAEstimatedPose, cameraBEstimatedPose);
-
-      // if (poseAgreement) {
-      //   m_swerveDriveTrain.addVisionMeasurement(
-      //       cameraAEstimatedPose.toPose2d(), cameraATimestamp);
-      //   m_swerveDriveTrain.addVisionMeasurement(
-      //       cameraBEstimatedPose.toPose2d(), cameraBTimestamp);
-      // }
-
-      // if (getTargetAmount(aprilTagLimelightCameraB) >= 2) {
-      //   m_swerveDriveTrain.addVisionMeasurement(
-      //       cameraBEstimatedPose, cameraBTimestamp);
-      // }
+    else {
+      DriverStation.reportWarning("LimelightA got vison pose", false);
+      estPoseLLA.set(limelightMeasurementCam1.pose);
+      estTimeStamp.set(limelightMeasurementCam1.timestampSeconds);
     }
-
-    switch (trackingState) {
-      case BRANCH:
-        updateAngleToBranch();
-        break;
-      case NONE:
-      default:
-        break;
+  
+    // if our angular velocity is greater than 360 degrees per second, ignore vision updates
+    // if(Math.abs(m_swerveDriveTrain.getPigeon2().getAngularVelocityZDevice().getValueAsDouble()) > 720)
+    // {
+    //   doRejectUpdateLLA = true;
+    // }
+    if(limelightMeasurementCam1.tagCount == 0)
+    {
+      doRejectUpdateLLA = true;
+    }
+    if(limelightMeasurementCam1.tagCount >= 1)
+    {
+      doRejectUpdateLLA = false;
+    }
+    if(!doRejectUpdateLLA) {
+      m_swerveDriveTrain.addVisionMeasurement(
+        limelightMeasurementCam1.pose,
+       limelightMeasurementCam1.timestampSeconds
+      );
     }
 
-    m_limelightAPositionPub.set(
-        new double[] {
-          VISION.LOCALIZER_CAMERA_POSITION[0].getTranslation().getX(),
-          VISION.LOCALIZER_CAMERA_POSITION[0].getTranslation().getY(),
-          VISION.LOCALIZER_CAMERA_POSITION[0].getTranslation().getZ(),
-        });
-
-    m_limelightBPositionPub.set(
-        new double[] {
-          VISION.LOCALIZER_CAMERA_POSITION[1].getTranslation().getX(),
-          VISION.LOCALIZER_CAMERA_POSITION[1].getTranslation().getY(),
-          VISION.LOCALIZER_CAMERA_POSITION[1].getTranslation().getZ(),
-        });
-
-    // This method will be called once per scheduler run
-    updateSmartDashboard();
-    if (ROBOT.logMode.get() <= ROBOT.LOG_MODE.NORMAL.get()) updateLog();
+    // limelight b
+    LimelightHelpers.SetRobotOrientation("limelight-b", m_swerveDriveTrain.getState().Pose.getRotation().getDegrees(), 0, 0, 0, 0, 0);
```

Comment:
Same as this comment: https://github.com/4201VitruvianBots/Reefscape2025/pull/122/files#r1986144962

Let's add this line here as well:
```
LimelightHelpers.SetIMUMode("limelight-b", 1);
```